ui/src/heap/HeapChannel.tsx

```diff
@@ -138,41 +140,90 @@ function HeapChannel({ title }: ViewProps) {
   });
 
   const renderCurio = useCallback(
-    (curio: HeapCurio, time: bigInt.BigInteger) => (
-      <div
-        key={time.toString()}
-        tabIndex={0}
-        className="cursor-pointer"
-        onClick={() => navigateToDetail(time)}
-      >
-        {displayMode === GRID ? (
-          <div className="aspect-h-1 aspect-w-1">
-            <HeapBlock curio={curio} time={time.toString()} />
-          </div>
-        ) : (
-          <HeapRow key={time.toString()} curio={curio} time={time.toString()} />
-        )}
-      </div>
-    ),
-    [displayMode, navigateToDetail]
+    (i: number, curio: HeapCurio, time: bigInt.BigInteger) =>
```

Comment:
I wonder if the `Header` component that virtuoso has would work here instead of having to fake it out

Reply:
Tried it, it places it outside of the grid (above it).